### PR TITLE
New project and device: Qualcomm/Dragonboard

### PR DESCRIPTION
**Pull request by lrusak**
No reason to hold this back anymore. I've been maintaining this branch for 2 years now.

I didn't add any of the HW accell patches as that can come later.

Some project/device options may need to be cleaned up but I'll leave others to point that out.

### CODE REVIEW

File: projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="mkbootimg"
+PKG_VERSION="c2bfa15"
```
Comment (MilhouseVH) [inline projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk]: Full rev `c2bfa15aa5bfc4e3f1b32de28e327b80b37db1cf` (sha256: `c980a0c59eed3635b5c9761781b12047256b48fa0a2e6f3ce639c5db4629290a`)?

This is also available with tag `1.6.0` (sha256: `7930b7dba25e5b4a39e6d19ccce9005e17c98a2b6c9e1aa6408a88c716755aaf`)

File: projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```
Comment (MilhouseVH) [inline projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk]: `or-later`?

File: projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="mkbootimg"
+PKG_VERSION="c2bfa15"
+PKG_SHA256="4367267088dbcdb4bd840fa64f836ddb6157f3b394a5d6f663e1b599162499cb"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://source.codeaurora.org/quic/kernel/skales/"
+PKG_URL="https://source.codeaurora.org/quic/kernel/skales/snapshot/$PKG_VERSION.tar.xz"
+PKG_SOURCE_DIR="$PKG_VERSION*"
```
Comment (MilhouseVH) [inline projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk]: You can drop `PKG_SOURCE_DIR` now

File: packages/linux-firmware/firmware-dragonboard/package.mk [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```
Comment (CvH) [inline packages/linux-firmware/firmware-dragonboard/package.mk]:
```suggestion
# SPDX-License-Identifier: GPL-2.0
```

File: projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk (hunk 2 above)
Comment (CvH) [inline projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk]:
```suggestion
# SPDX-License-Identifier: GPL-2.0
```

File: projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="mkbootimg"
+PKG_VERSION="c2bfa15"
+PKG_SHA256="4367267088dbcdb4bd840fa64f836ddb6157f3b394a5d6f663e1b599162499cb"
+PKG_ARCH="any"
```
Comment (CvH) [inline projects/Qualcomm/devices/Dragonboard/packages/mkbootimg/package.mk]:
```suggestion
```

File: projects/Qualcomm/devices/Dragonboard/README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+## Install Instructions
+
```
Comment (CvH) [inline projects/Qualcomm/devices/Dragonboard/README.md]: could you add something like https://github.com/LibreELEC/LibreELEC.tv/blob/master/projects/Rockchip/devices/RK3328/README.md that we know how to build it.

Comment (lrusak) [reply projects/Qualcomm/devices/Dragonboard/README.md]: It's no different then any other platform to build. Only the install is different and that's why I documented it.

**Comment by lrusak** [comment]
updated

**Comment by MilhouseVH** [comment]
Many thanks!

**Pull request merged**